dora/shell/src/main/java/alluxio/cli/fsadmin/command/NodesCommand.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.cli.Command;
+import alluxio.cli.fsadmin.nodes.WorkerStatusCommand;
+import alluxio.conf.AlluxioConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+/**
+ * Command to get or administrate worker nodes status.
+ */
+public class NodesCommand extends AbstractFsAdminCommand {
```

Comment:
would be nice if we can have an integration test

Reply:
The bug fix for the etcd client connection and a new CLI command should be two separate PRs.

Reply:
ok let me carve out this cmd piece and add integration test over there